### ISSUE TITLE
RuntimeError: Unsuccessful TensorSliceReader constructor: Failed to find any matching files for /venvs/anaconda/envs/clair3/bin/models/r941_prom_sup_g506//pileup

### ISSUE THREAD

**Issue by billytcl** [HKU-BAL/Clair3#70]
I'm trying to run some variant calling on cDNA, but I'm getting the following repeated error. I'm not sure if it has anything to do with the splicing itself or if it's another issue:

Here's the log:

```
[INFO] CLAIR3 VERSION: v0.1-r8
[INFO] BAM FILE PATH: /mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/merged.sorted.bam
[INFO] REFERENCE FILE PATH: /mnt/ix1/Resources/GenomeRef/Homo_sapiens/Ensembl/GRCh38_no_alt/Sequence/WholeGenomeFasta/hs38_naa.fna
[INFO] MODEL PATH: /venvs/anaconda/envs/clair3/bin/models/r941_prom_sup_g506/
[INFO] OUTPUT FOLDER: /mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/clair3_out
[INFO] PLATFORM: ont
[INFO] THREADS: 20
[INFO] BED FILE PATH: /mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/STAMP-foundation.panel.bed
[INFO] VCF FILE PATH: EMPTY
[INFO] CONTIGS: EMPTY
[INFO] CONDA PREFIX: /venvs/anaconda/envs/clair3
[INFO] SAMTOOLS PATH: samtools
[INFO] PYTHON PATH: python3
[INFO] PYPY PATH: pypy3
[INFO] PARALLEL PATH: parallel
[INFO] WHATSHAP PATH: whatshap
[INFO] CHUNK SIZE: 5000000
[INFO] FULL ALIGN PROPORTION: 0.7
[INFO] FULL ALIGN REFERENCE PROPORTION: 0.1
[INFO] ENABLE FILEUP ONLY CALLING: False
[INFO] ENABLE FAST MODE CALLING: False
[INFO] ENABLE CALLING SNP CANDIDATES ONLY: False
[INFO] ENABLE PRINTING REFERENCE CALLS: False
[INFO] ENABLE OUTPUT GVCF: False
[INFO] ENABLE HAPLOID PRECISE MODE: False
[INFO] ENABLE HAPLOID SENSITIVE MODE: False
[INFO] ENABLE INCLUDE ALL CTGS CALLING: False
[INFO] ENABLE NO PHASING FOR FULL ALIGNMENT: False
[INFO] ENABLE REMOVING INTERMEDIATE FILES: False
[INFO] ENABLE PHASING VCF OUTPUT: False

+ /venvs/anaconda/envs/clair3/bin/scripts/clair3.sh --bam_fn /mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/merged.sorted.bam --ref_fn /mnt/ix1/Resources/GenomeRef/Homo_sapiens/Ensembl/GRCh38_no_alt/Sequence/WholeGenomeFasta/hs38_naa.fna --threads 20 --model_path /venvs/anaconda/envs/clair3/bin/models/r941_prom_sup_g506/ --platform ont --output /mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/clair3_out --bed_fn=/mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/STAMP-foundation.panel.bed --vcf_fn=EMPTY --ctg_name=EMPTY --sample_name=SAMPLE --chunk_num=0 --chunk_size=5000000 --samtools=samtools --python=python3 --pypy=pypy3 --parallel=parallel --whatshap=whatshap --qual=2 --var_pct_full=0.7 --ref_pct_full=0.1 --snp_min_af=0 --indel_min_af=0 --pileup_only=False --gvcf=False --fast_mode=False --call_snp_only=False --print_ref_calls=False --haploid_precise=False --haploid_sensitive=False --include_all_ctgs=False --no_phasing_for_fa=False --pileup_model_prefix=pileup --fa_model_prefix=full_alignment --remove_intermediate_dir=False --enable_phasing=False
[INFO] Check environment variables
[INFO] Create folder /mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/clair3_out/log
[INFO] Create folder /mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/clair3_out/tmp
[INFO] Create folder /mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/clair3_out/tmp/split_beds
[INFO] Create folder /mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/clair3_out/tmp/pileup_output
[INFO] Create folder /mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/clair3_out/tmp/merge_output
[INFO] Create folder /mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/clair3_out/tmp/phase_output
[INFO] Create folder /mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/clair3_out/tmp/gvcf_tmp_output
[INFO] Create folder /mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/clair3_out/tmp/full_alignment_output
[INFO] Create folder /mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/clair3_out/tmp/phase_output/phase_vcf
[INFO] Create folder /mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/clair3_out/tmp/phase_output/phase_bam
[INFO] Create folder /mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/clair3_out/tmp/full_alignment_output/candidate_bed
[INFO] --include_all_ctgs not enabled, use chr{1..22,X,Y} and {1..22,X,Y} by default
[INFO] Call variant in contigs: chr1 chr2 chr3 chr4 chr5 chr6 chr7 chr8 chr9 chr10 chr11 chr12 chr13 chr14 chr15 chr16 chr17 chr18 chr19 chr20 chr21 chr22 chrX
[INFO] Chunk number for each contig: 50 49 40 39 37 35 32 30 28 27 28 27 23 22 21 19 17 17 12 13 10 11 32
[INFO] 1/7 Call variants using pileup model
[INFO] Delay 0 seconds before starting variant calling ...
[mpileup] 1 samples in 1 input files
Traceback (most recent call last):
  File "/venvs/anaconda/envs/clair3/lib/python3.6/site-packages/tensorflow/python/training/py_checkpoint_reader.py", line 95, in NewCheckpointReader
    return CheckpointReader(compat.as_bytes(filepattern))
RuntimeError: Unsuccessful TensorSliceReader constructor: Failed to find any matching files for /venvs/anaconda/envs/clair3/bin/models/r941_prom_sup_g506//pileup

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/venvs/anaconda/envs/clair3/bin/clair3/../clair3.py", line 91, in <module>
    main()
  File "/venvs/anaconda/envs/clair3/bin/clair3/../clair3.py", line 85, in main
    submodule.main()
  File "/venvs/anaconda/envs/clair3/bin/clair3/CallVariants.py", line 1803, in main
    Run(args)
  File "/venvs/anaconda/envs/clair3/bin/clair3/CallVariants.py", line 205, in Run
    call_variants(args=args, output_config=output_config, output_utilities=output_utilities)
  File "/venvs/anaconda/envs/clair3/bin/clair3/CallVariants.py", line 1379, in call_variants
    m.load_weights(args.chkpnt_fn)
  File "/venvs/anaconda/envs/clair3/lib/python3.6/site-packages/tensorflow/python/keras/engine/training.py", line 250, in load_weights
    return super(Model, self).load_weights(filepath, by_name, skip_mismatch)
  File "/venvs/anaconda/envs/clair3/lib/python3.6/site-packages/tensorflow/python/keras/engine/network.py", line 1231, in load_weights
    py_checkpoint_reader.NewCheckpointReader(filepath)
  File "/venvs/anaconda/envs/clair3/lib/python3.6/site-packages/tensorflow/python/training/py_checkpoint_reader.py", line 99, in NewCheckpointReader
    error_translator(e)
  File "/venvs/anaconda/envs/clair3/lib/python3.6/site-packages/tensorflow/python/training/py_checkpoint_reader.py", line 35, in error_translator
    raise errors_impl.NotFoundError(None, None, error_message)
tensorflow.python.framework.errors_impl.NotFoundError: Unsuccessful TensorSliceReader constructor: Failed to find any matching files for /venvs/anaconda/envs/clair3/bin/models/r941_prom_sup_g506//pileup
[INFO] Delay 4 seconds before starting variant calling ...
[mpileup] 1 samples in 1 input files
Traceback (most recent call last):
  File "/venvs/anaconda/envs/clair3/lib/python3.6/site-packages/tensorflow/python/training/py_checkpoint_reader.py", line 95, in NewCheckpointReader
    return CheckpointReader(compat.as_bytes(filepattern))
RuntimeError: Unsuccessful TensorSliceReader constructor: Failed to find any matching files for /venvs/anaconda/envs/clair3/bin/models/r941_prom_sup_g506//pileup

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/venvs/anaconda/envs/clair3/bin/clair3/../clair3.py", line 91, in <module>
    main()
  File "/venvs/anaconda/envs/clair3/bin/clair3/../clair3.py", line 85, in main
    submodule.main()
  File "/venvs/anaconda/envs/clair3/bin/clair3/CallVariants.py", line 1803, in main
    Run(args)
  File "/venvs/anaconda/envs/clair3/bin/clair3/CallVariants.py", line 205, in Run
    call_variants(args=args, output_config=output_config, output_utilities=output_utilities)
  File "/venvs/anaconda/envs/clair3/bin/clair3/CallVariants.py", line 1379, in call_variants
    m.load_weights(args.chkpnt_fn)
  File "/venvs/anaconda/envs/clair3/lib/python3.6/site-packages/tensorflow/python/keras/engine/training.py", line 250, in load_weights
    return super(Model, self).load_weights(filepath, by_name, skip_mismatch)
  File "/venvs/anaconda/envs/clair3/lib/python3.6/site-packages/tensorflow/python/keras/engine/network.py", line 1231, in load_weights
    py_checkpoint_reader.NewCheckpointReader(filepath)
  File "/venvs/anaconda/envs/clair3/lib/python3.6/site-packages/tensorflow/python/training/py_checkpoint_reader.py", line 99, in NewCheckpointReader
    error_translator(e)
  File "/venvs/anaconda/envs/clair3/lib/python3.6/site-packages/tensorflow/python/training/py_checkpoint_reader.py", line 35, in error_translator
    raise errors_impl.NotFoundError(None, None, error_message)
tensorflow.python.framework.errors_impl.NotFoundError: Unsuccessful TensorSliceReader constructor: Failed to find any matching files for /venvs/anaconda/envs/clair3/bin/models/r941_prom_sup_g506//pileup

...

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/venvs/anaconda/envs/clair3/bin/clair3/../clair3.py", line 91, in <module>
    main()
  File "/venvs/anaconda/envs/clair3/bin/clair3/../clair3.py", line 85, in main
    submodule.main()
  File "/venvs/anaconda/envs/clair3/bin/clair3/CallVariants.py", line 1803, in main
    Run(args)
  File "/venvs/anaconda/envs/clair3/bin/clair3/CallVariants.py", line 205, in Run
    call_variants(args=args, output_config=output_config, output_utilities=output_utilities)
  File "/venvs/anaconda/envs/clair3/bin/clair3/CallVariants.py", line 1379, in call_variants
    m.load_weights(args.chkpnt_fn)
  File "/venvs/anaconda/envs/clair3/lib/python3.6/site-packages/tensorflow/python/keras/engine/training.py", line 250, in load_weights
    return super(Model, self).load_weights(filepath, by_name, skip_mismatch)
  File "/venvs/anaconda/envs/clair3/lib/python3.6/site-packages/tensorflow/python/keras/engine/network.py", line 1231, in load_weights
    py_checkpoint_reader.NewCheckpointReader(filepath)
  File "/venvs/anaconda/envs/clair3/lib/python3.6/site-packages/tensorflow/python/training/py_checkpoint_reader.py", line 99, in NewCheckpointReader
    error_translator(e)
  File "/venvs/anaconda/envs/clair3/lib/python3.6/site-packages/tensorflow/python/training/py_checkpoint_reader.py", line 35, in error_translator
    raise errors_impl.NotFoundError(None, None, error_message)
tensorflow.python.framework.errors_impl.NotFoundError: Unsuccessful TensorSliceReader constructor: Failed to find any matching files for /venvs/anaconda/envs/clair3/bin/models/r941_prom_sup_g506//pileup
Traceback (most recent call last):
  File "/venvs/anaconda/envs/clair3/bin/clair3/../clair3.py", line 91, in <module>
    main()
  File "/venvs/anaconda/envs/clair3/bin/clair3/../clair3.py", line 85, in main
    submodule.main()
  File "/venvs/anaconda/envs/clair3/bin/preprocess/CreateTensorPileup.py", line 552, in main
    CreateTensorPileup(args)
  File "/venvs/anaconda/envs/clair3/bin/preprocess/CreateTensorPileup.py", line 413, in CreateTensorPileup
    tensor_can_fp.stdin.write(l)
BrokenPipeError: [Errno 32] Broken pipe

real    4m23.407s
user    25m34.373s
sys     4m59.363s
[WARNING] No vcf file found with prefix:/mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/clair3_out/tmp/pileup_output/pileup, output empty vcf file
[WARNING] Copying pileup.vcf.gz to /mnt/ix1/Seq_Runs/20211005_MIN_1086/vcf_selected/clair3_out/merge_output.vcf.gz
[INFO] Exit in pileup variant calling

real    4m53.091s
user    25m36.328s
sys     5m0.092s
```

**Comment by zhengzhenxian** [comment]
@billytcl Could you try to replace `--model_path="${CONDA_PREFIX}/bin/models/r941_prom_sup_g506/"` by `--model_path="${CONDA_PREFIX}/bin/models/r941_prom_sup_g506"` (remove the last slash)? I think it would work then.
